Source/SwiftLintFramework/Rules/Style/FileTypesOrderRule.swift

```diff
@@ -98,7 +111,7 @@ public struct FileTypesOrderRule: ConfigurationProviderRule, OptInRule {
         return dict.substructure.filter { substructure in
             guard let kind = substructure.kind else { return false }
             return substructure.offset != mainTypeSubstructure.offset &&
-                kind.contains(SwiftDeclarationKind.extension.rawValue)
+            kind.contains(SwiftDeclarationKind.extension.rawValue)
```

Comment:
Why is the indentation removed?

Reply:
I used Xcode indentation to reindent the file, I reverted it now.